frontend/src/component/common/Input/Input.styles.ts

```diff
@@ -3,7 +3,7 @@ import { makeStyles } from 'tss-react/mui';
 export const useStyles = makeStyles()(theme => ({
     helperText: {
         position: 'absolute',
-        top: '35px',
+        bottom: '-1rem',
```

Comment:
For the most part, this should give us the same behaviour as the previous setting, except now it works on every type of field, no matter the height (e.g. multiline fields) instead of breaking.